.github/workflows/test-mlperf-inference-abtf-poc.yml

```diff
@@ -47,6 +52,61 @@ jobs:
         pip install cmind
         cm pull repo --url=${{ github.event.pull_request.head.repo.html_url }} --checkout=${{ github.event.pull_request.head.ref }}
         cm pull repo mlcommons@cm4abtf --branch=poc
+  
+    - name: Install Docker on macos
+      if: runner.os == 'macOS-deactivated'
+      run: |
+        brew update
+        brew install --cask docker
+
+    - name: Start Docker Daemon on macos
+      if: runner.os == 'macOS-deactivated'
+      run: |
+        open /Applications/Docker.app
```

Comment:
When the docker application starts up for the first time, it asks for accepting the aggreements then only proceeds to starting the docker daemon. Maybe we have to find some better way?
![Screenshot from 2024-11-06 18-52-23](https://github.com/user-attachments/assets/bd08c10c-7b3e-488d-9fda-48310a967954)